web/src/components/PublicationsPrevNext/PublicationsPrevNext.module.scss

```diff
@@ -0,0 +1,3 @@
+.hr {
+  margin-top: 1rem !important;
+}
```

Comment:
I don't think this is being used in PublicationsPrevNext - did you add/leave it in by accident? Are we using this class in another component?